content/blog/anotherpost/index.md

```diff
@@ -0,0 +1,224 @@
+---
+path: "numtwo"
```

Comment:
The Slug should be formatted with a back slash -> `"/numtwo"` also please call it slug and not path.